File: backend/src/main/java/com/example/demo/entities/Customer.java
package com.example.demo.entities;

import java.sql.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@ToString
@Table(name = "customers")
public class Customer {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "customerId")
    private int customer_id;
    
    @Column(name = "fname")
    private String fname;
    
    @Column(name = "lname")
    private String lname;
    
    @Column(name = "license_no")
    private String license_no;
    
    @Column(name = "contact")
    private String contact;
    
    @Column(name = "emergency_contact")
    private String emergency_contact;
    
    @Column(name = "dob")
    private Date dob;
    
    @Column(name = "reg_date")
    private Date reg_date;//
    
    @Column(name = "pancard_no")
    private String pancard_no;
    
    @Column(name = "adhar_card")
    private String adhar_card;

    @ManyToOne
    @JoinColumn(name = "uid")
    private User user;

    @ManyToOne
    @JoinColumn(name = "area_id")
    private Area area;
    
    @Column(name="address")
    private String address;
    
    @Column(name="email_id")
    private String email_id;
    
    
}

File: dot.net/BookMyCar/Models/Customer.cs
using System;
using System.Collections.Generic;

namespace BookMyCar.Models;

public partial class Customer
{
    public int CustomerId { get; set; }

    public string Fname { get; set; } = null!;

    public string Lname { get; set; } = null!;

    public string LicenseNo { get; set; } = null!;

    public string Contact { get; set; } = null!;

    public string EmergencyContact { get; set; } = null!;

    public DateOnly Dob { get; set; }

    public DateOnly? RegDate { get; set; }

    public string PancardNo { get; set; } = null!;

    public string AdharCard { get; set; } = null!;

    public int? Uid { get; set; }

    public int? AreaId { get; set; }

    public string? Address { get; set; }

    public string? EmailId { get; set; }

    public virtual Area? Area { get; set; }

    public virtual ICollection<BookingRequest> BookingRequests { get; set; } = new List<BookingRequest>();

    public virtual ICollection<Feedback> Feedbacks { get; set; } = new List<Feedback>();

    public virtual User? UidNavigation { get; set; }
}
